corehq/ex-submodules/casexml/apps/case/tests/util.py

```diff
@@ -105,20 +105,24 @@ def check_xml_line_by_line(test_case, expected, actual):
 
 
 def assert_user_has_case(testcase, user, case_id, **kwargs):
+    """DEPRECATED use assertIn(case_id, <MockDevice>.sync().cases)"""
```

Comment:
worth using

```python
warnings.warn("use assertIn(case_id, <MockDevice>.sync().cases)", DeprecationWarning)
```

or changing name to `deprecated_assert_user_has_case` instead?